Added property to evaulate platform theme mode (iOS, web, Windows) (#4408)
beeware/toga: changes/4408.feature.md

> Added property to evaluate platform theme mode on iOS, web, and Windows.

Reviewer: ```suggestion
Dark mode can now be detected on iOS, web, and Windows.
```
